wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/exception/ExceptionConstants.java

```diff
@@ -9,7 +9,17 @@ public class ExceptionConstants {
     private static final String CODE_SUCHKRITERIEN_UNVOLLSTAENDIG = "901";
     private static final String MSG_SUCHKRITERIEN_UNVOLLSTAENDIG = "Fehler beim Laden: Suchkriterien unvollständig.";
 
+    private static final String CODE_PARAMS_UNVOLLSTAENDIG = "902";
+    private static final String MSG_PARAMS_UNVOLLSTAENDIG = "Fehler beim Speichern: Parameter unvollständig.";
+
+    private static final String CODE_UNSAVEABLE = "903";
+    private static final String MSG_UNSAVEABLE = "Fehler beim speichern: Daten konnten nich gespeichert werden.";
```

Comment:
I can't check the correctness of these without knowing where they are documented.

Reply:
we talked